From 0895c075196923c89227a1cf4a31fc8eca553049 Mon Sep 17 00:00:00 2001
From: Raviteja Sunkara <raviteja@codeaurora.org>
Date: Fri, 26 Apr 2013 21:59:16 +0530
Subject: [PATCH] remove the limit in the query of patch sets by revision.

Currently there a limit of two for the results of query
based on revision. The reason to put this limit when the
query was written is unknown.
Since, there can be many changes that correspond to a
commit and all the changes should be provided to user,
this commit removes that limit.

Change-Id: I49401e37b70caa448176d6394bc0f9159ebe2fed
---
 .../gerrit/reviewdb/server/PatchSetAccess.java     |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gerrit-reviewdb/src/main/java/com/google/gerrit/reviewdb/server/PatchSetAccess.java b/gerrit-reviewdb/src/main/java/com/google/gerrit/reviewdb/server/PatchSetAccess.java
index 7e0b90c..703edbb 100644
--- a/gerrit-reviewdb/src/main/java/com/google/gerrit/reviewdb/server/PatchSetAccess.java
+++ b/gerrit-reviewdb/src/main/java/com/google/gerrit/reviewdb/server/PatchSetAccess.java
@@ -30,10 +30,10 @@ public interface PatchSetAccess extends Access<PatchSet, PatchSet.Id> {
   @Query("WHERE id.changeId = ? ORDER BY id.patchSetId")
   ResultSet<PatchSet> byChange(Change.Id id) throws OrmException;
 
-  @Query("WHERE revision = ? LIMIT 2")
+  @Query("WHERE revision = ?")
   ResultSet<PatchSet> byRevision(RevId rev) throws OrmException;
 
-  @Query("WHERE revision >= ? AND revision <= ? LIMIT 2")
+  @Query("WHERE revision >= ? AND revision <= ?")
   ResultSet<PatchSet> byRevisionRange(RevId reva, RevId revb)
       throws OrmException;
 }
-- 
1.7.8.3

